### PR TITLE
Minio increase mem limit

### PR DESCRIPTION
**Pull request by dekiel**
Increasing memory limits due to OOM events.

kyma-integration-production-gardener-azure #1361934901653278720

Warning  SystemOOM                26m                kubelet, shoot--kyma-prow--grdo4epnf-cpu-worker-z1-76789-fpv8b          System OOM encountered, victim process: minio, pid: 88813
  Warning  OOMKilling               26m                kernel-monitor, shoot--kyma-prow--grdo4epnf-cpu-worker-z1-76789-fpv8b   Memory cgroup out of memory: Killed process 88813 (minio) total-vm:1026044kB, **anon-rss:257032kB, file-rss:29576kB**, shmem-rss:0kB, UID:1000 pgtables:736kB oom_score_adj:999

kyma-integration-evaluation-gardener-azure #1361844301939609600

Warning  OOMKilling               14m                kernel-monitor, shoot--kyma-prow--grd2y5qav-cpu-worker-z1-5ffd4-gpnnx   Memory cgroup out of memory: Killed process 37653 (minio) total-vm:1025020kB, **anon-rss:258636kB, file-rss:30412kB**, shmem-rss:0kB, UID:1000 pgtables:684kB oom_score_adj:999
  Warning  SystemOOM                14m                kubelet, shoot--kyma-prow--grd2y5qav-cpu-worker-z1-5ffd4-gpnnx          System OOM encountered, victim process: minio, pid: 37653

There is no memory limit set for minio container resources/rafter/charts/controller-manager/charts/minio/templates/deployment.yaml, only requests are provided from values.yaml.

```
## Configure resource requests and limits
## ref: http://kubernetes.io/docs/user-guide/compute-resources/
##
resources:
  requests:
    memory: 256Mi
    cpu: 250m
```

According to limits set here resources/cluster-essentials/templates/limit-range.yaml, default memory limit is 96Mi.

See https://github.com/kyma-project/test-infra/issues/3300

### CODE REVIEW

File: resources/rafter/charts/controller-manager/charts/minio/values.yaml [hunk 1]
```diff
@@ -168,6 +168,8 @@ resources:
   requests:
     memory: 256Mi
     cpu: 250m
+  limits:
+    memory: 320Mi
```
Comment (tgorgol) [inline resources/rafter/charts/controller-manager/charts/minio/values.yaml:172]: This values are overridden by rafter chart's values, please change the value there:

https://github.com/kyma-project/kyma/blob/903a710158685984a97e6182baf17d315590791c/resources/rafter/values.yaml#L254

Comment (Halamix2) [reply resources/rafter/charts/controller-manager/charts/minio/values.yaml]: Fixed

**Review by tgorgol — APPROVED** [review]

**Pull request merged**